phpunit.xml.dist

```diff
@@ -17,8 +21,14 @@
         <whitelist>
             <directory>.</directory>
             <exclude>
-                <directory>Resources</directory>
+                <directory>./Resources</directory>
+                <directory>./Tests</directory>
```

Comment:
Just to clarify: What is this good for? Doesn't this mean, the `Tests` directory gets ignored?